main.js

```diff
@@ -0,0 +1,130 @@
+const myLibrary = [];
+
+const book = (author, title, pages, read) => {
+  const getBooks = () => myLibrary;
+
+  const addBookToLibrary = () => {
+    myLibrary.push({
+      author, title, pages, read,
+    });
+  };
+
+  return {
+    addBookToLibrary, author, title, pages, read, getBooks,
+  };
+};
```

Comment:
here's a correct way to write a constructor function:
```
function Book(name, author, pages, read) {
    this.name = name;
    this.author = author;
    this.pages = pages;
    this.read = read;
}
```

Reply:
done